core/stat/stat_slot.go

```diff
@@ -60,6 +95,12 @@ func (s *Slot) OnCompleted(ctx *base.EntryContext) {
 	if ctx.Resource.FlowType() == base.Inbound {
 		s.recordCompleteFor(InboundNode(), ctx.Input.BatchCount, rt, ctx.Err())
 	}
+
+	completeCounter.Add(float64(ctx.Input.BatchCount), ctx.Resource.Name())
+	if ctx.Err() != nil {
```

Comment:
When ctx.isBlocked()=true and ctx.err ()!= nil, errorCounter can't count

Reply:
Currently removed.